csrc/multidevice/communicator.cpp

```diff
@@ -14,7 +14,7 @@
 #ifdef USE_C10D_NCCL
 #include <torch/csrc/distributed/c10d/ProcessGroupNCCL.hpp>
 #endif
-#ifdef USE_C10D_UCC
+#if defined(USE_C10D_UCC) && defined(NVFUSER_BUILD_WITH_UCC)
```

Comment:
This looks ugly. But I understand that we are just doing the minimal to get CI green so it's fine for now.

Reply:
😟